### PR TITLE
Create python-package-conda.yml

### PR DESCRIPTION
**Pull request by bartix3**

### CODE REVIEW

**Comment by kovidgoyal** [comment]
What is this for?

File: .github/workflows/python-package-conda.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+    - name: Add conda to system path
+      run: |
+        # $CONDA is an environment variable pointing to the root of the miniconda directory
+        echo $CONDA/bin >> $GITHUB_PATH
+    - name: Install dependencies
+      run: |
+        conda env update --file environment.yml --name base
```
Comment (eli-schwartz) [inline .github/workflows/python-package-conda.yml:23]: environment.yml file doesn't exist.

File: .github/workflows/python-package-conda.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      run: |
+        conda install flake8
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
+      run: |
+        conda install pytest
+        pytest
```
Comment (eli-schwartz) [inline .github/workflows/python-package-conda.yml:34]: The testsuite doesn't use pytest.

**Review by eli-schwartz — COMMENTED** [review]
@kovidgoyal it's clearly just spam.

My theory based on github profile examination is that the spam in question is the result of using ChatGPT to try to get contribution count up, resulting in changes that superficially look meaningful but actually are not.

**Comment by kovidgoyal** [comment]
Sigh. Thanks @eli-schwartz

**Comment by bartix3** [comment]
I'm very sorry. I'm just learning Github. And I mixed things up a bit. I in
no way intended to spam or had any malicious intent. I'm sorry once again,
best regards, Bartek


W dniu niedz., 17.03.2024 o 04:46 Kovid Goyal ***@***.***>
napisał(a):

**Comment by bartix3** [comment]
I'm very sorry. I'm just learning Github. And I mixed things up a bit. I in
no way intended to spam or had any malicious intent. I'm sorry once again,
best regards, Bartek


W dniu niedz., 17.03.2024 o 02:25 Eli Schwartz ***@***.***>
napisał(a):

**Comment by kovidgoyal** [comment]
no worries.

**Pull request closed without merging**